pkgs/applications/networking/cluster/waypoint/default.nix

```diff
@@ -1,46 +1,60 @@
-{ lib, buildGoModule, fetchFromGitHub, go-bindata }:
-
+{ buildGoPackage, buildGoModule, fetchFromGitHub, lib }:
+
+let
+  # nixpkgs go-bindata causes issues with the waypoint-entrypoint
+  go-bindata = buildGoPackage rec {
+    pname = "go-bindata";
```

Comment:
I just merged the PR that adds go-bindata globally. Can we use that here?

Reply:
Yup its the same version. Ill rebase

Reply:
rebased